components/dashboard/src/whatsnew/MigrationPage.tsx

```diff
@@ -34,6 +90,8 @@ export function MigrationPage() {
                         isValid={true}
                         isSaving={migrateUsers.isLoading}
                         onSubmit={migrateUsers.mutateAsync}
+                        onCancel={skipForNow}
+                        cancelButtonText="Ask me later"
                     >
                         <Heading3>What's different?</Heading3>
                         <p className="text-gray-500 text-base mb-4">
```

Comment:
**issue(non-blocking)**: Minor character issue—could we replace the `-&gt;` character with `→` in the help link?